lib/puma/const.rb

```diff
@@ -176,6 +176,7 @@ module Const
     PORT_443 = "443".freeze
     LOCALHOST = "localhost".freeze
     LOCALHOST_IP = "127.0.0.1".freeze
+    UNSPECIFIED_IP = "0.0.0.0".freeze
```

Comment:
I wonder if we might have trouble here with ipv6 setups.

Reply:
I guess we could ask the socket for its type and then return `0.0.0.0` or `::` depending? I can make that change if you'd like.

Reply:
@Roguelazer Please do - I think it makes sense.